packages/cli/test/integration/generators/interceptor.integration.js

```diff
@@ -0,0 +1,129 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const path = require('path');
+const assert = require('yeoman-assert');
+const testlab = require('@loopback/testlab');
+
+const TestSandbox = testlab.TestSandbox;
+
+const generator = path.join(__dirname, '../../../generators/interceptor');
+const SANDBOX_FILES = require('../../fixtures/interceptor').SANDBOX_FILES;
+const testUtils = require('../../test-utils');
+
+// Test Sandbox
+const SANDBOX_PATH = path.resolve(__dirname, '..', '.sandbox');
+const sandbox = new TestSandbox(SANDBOX_PATH);
+
+describe('lb4 interceptor', () => {
+  beforeEach('reset sandbox', async () => {
+    await sandbox.reset();
+  });
+
+  describe('valid generation of interceptors', () => {
+    it('generates a basic interceptor from command line arguments', async () => {
+      await testUtils
+        .executeGenerator(generator)
+        .inDir(SANDBOX_PATH, () =>
+          testUtils.givenLBProject(SANDBOX_PATH, {
+            additionalFiles: SANDBOX_FILES,
+          }),
+        )
+        .withArguments('myInterceptor');
+      verifyGeneratedScript();
+    });
+
+    it('generates a basic interceptor from CLI with group', async () => {
+      await testUtils
+        .executeGenerator(generator)
+        .inDir(SANDBOX_PATH, () =>
+          testUtils.givenLBProject(SANDBOX_PATH, {
+            additionalFiles: SANDBOX_FILES,
+          }),
+        )
+        .withArguments('myInterceptor --group myGroup');
+      verifyGeneratedScript(true, 'myGroup');
+    });
+
+    it('generates a interceptor from a config file', async () => {
+      await testUtils
+        .executeGenerator(generator)
+        .inDir(SANDBOX_PATH, () =>
+          testUtils.givenLBProject(SANDBOX_PATH, {
+            additionalFiles: SANDBOX_FILES,
+          }),
+        )
+        .withArguments('--config myinterceptorconfig.json');
+      verifyGeneratedScript();
+    });
+
+    it('generates a non-global interceptor from CLI', async () => {
+      await testUtils
+        .executeGenerator(generator)
+        .inDir(SANDBOX_PATH, () =>
+          testUtils.givenLBProject(SANDBOX_PATH, {
+            additionalFiles: SANDBOX_FILES,
+          }),
+        )
+        .withArguments('myInterceptor --no-global');
+      verifyGeneratedScript(false);
```

Comment:
Boolean arguments are evil. What does `false` means here? It's hard to tell from the name of the invoked function (`verifyGeneratedScript`). Please convert it to an options object, e.g.

```suggestion
      verifyGeneratedScript({global: false});
```